src/main/webapp/script.js

```diff
@@ -100,21 +100,84 @@ window.addEventListener('load', () => {
 
   getOptions();
 
-  function initCalendar() {
-    new tui.Calendar('#calendar', {
-      defaultView: 'week',
-      useCreationPopup: true,
-      useDetailPopup: true,
-      disableDblClick: true,
-      disableClick: true,
-      isReadOnly: true,
-      scheduleView: ['time'],
-      taskView: false,
-      week: {
-        workweek: true,
-      },
+  if (document.head.id == 'scheduler') {
+    function initCalendar() {
+      new tui.Calendar('#calendar', {
+        defaultView: 'week',
+        useCreationPopup: true,
+        useDetailPopup: true,
+        disableDblClick: true,
+        disableClick: true,
+        isReadOnly: true,
+        scheduleView: ['time'],
+        taskView: false,
+        week: {
+          workweek: true,
+        },
+      });
+    }
+
+    initCalendar();
+  } else {
+    document.getElementById('submit-plan').addEventListener('submit', () => {
+      /**
+       * Gets results from /planner servlet to results
+       */
+      async function getPlan() {
+        const courseContainer = document.getElementById('order-area');
+        createSpinner(courseContainer);
+        const response = await fetch(
+            `/planner?selectedClasses=${selected.toString()}&semesters=${
```

Comment:
To be safe, you should use `encodeURIComponent` for any url params. Otherwise, stray characters like `?` or `&` can mess up the url.

Reply:
I forgot - if this becomes a doPost, then you'll want to specify these arguments in the request body anyway, so this might be a moot point.